backend/.env.example

```diff
@@ -29,8 +29,8 @@ BLOCKCHAIN_REST_API_URL="http://localhost:5051"
 BLOCKCHAIN_API_SECRET="ef0b48e41f78d3ae85b1379b386f1bca"
 
 # Blockchain explorer to link to. Top for mainnet, bottom for testnet.
-# EXPLORER_URL="https://explorer.zcha.in/"
-EXPLORER_URL="https://testnet.zcha.in/"
+# EXPLORER_URL="https://chain.so/tx/ZECTEST/<txid>"
```

Comment:
Let's replace with `https://explorer.zcha.in/transactions/<txid>` so we don't mistake this when we change to mainnet.